lib/tasks/dev.rake

```diff
@@ -146,7 +152,7 @@ namespace :dev do
                 else
                   success = true
                 end
-                sleep(usps_request_delay_ms / 1000.0) if usps_request_delay_ms
+                SampleDataCreator.delay(usps_request_delay_ms / 1000.0) if usps_request_delay_ms
```

Comment:
What if we called `Kernel.sleep` directly, and stubbed that in tests? Then we don't need to define the one-off class, and can stub that instead?
```suggestion
                Kernel.sleep(usps_request_delay_ms / 1000.0) if usps_request_delay_ms
```

Reply:
Hm, I'm open to it. I guess I wondered if that would be prone to the same sort of flakiness (are `Kernel.sleep` and `sleep` effectively the same method?), or at least if any other unrelated code would be calling `Kernel.sleep` as well, since the assertions here are very strict about what's an allowed call.

Reply:
I'm hoping that no? that by harcoding a specific static object, `Kernel::sleep` we'd avoid the same ambiguity of "which instance was `sleep` called on" when we call the `sleep` that's mixed in to every object (`Kernel#sleep`)

Reply:
Let's give it a shot! Updated in 1de8106f0.